home-assistant-plugin/custom_components/light/xknx.py

```diff
@@ -80,6 +91,16 @@ def async_add_entities_discovery(hass, discovery_info, async_add_entities):
 def async_add_entities_config(hass, config, async_add_entities):
     """Set up light for KNX platform configured within platform."""
     import xknx
+
+    # switch group address for selected color_temperature_mode
+    ga_tw, ga_tw_s, ga_ct, ga_ct_s = None, None, None, None
```

Comment:
@farmio can you please use a more precise naming? (then the descriptions over it is also not needed anymore :) ).

This would not go through the HA review.